how-to/laravel.md

```diff
@@ -0,0 +1,653 @@
+---
+title: How to use PGroonga with Laravel
+---
+
+**# 01. Introduction**
```

Comment:
Could you use the title as the first heading?
Could you remove `**...**` markup from heading?

```suggestion
# How to use PGroonga with Laravel
```